testing/coordinator.go

```diff
@@ -174,7 +173,7 @@ func (coord *Coordinator) GetChain(chainID string) *TestChain {
 
 // GetChainID returns the chainID used for the provided index.
 func GetChainID(index int) string {
-	return ChainIDPrefix + strconv.Itoa(index)
+	return ChainIDPrefix + strconv.Itoa(index+1)
```

Comment:
Why +1?

Reply:
can you only +1 if `index` is 0 then?

Reply:
There will be duplicated ids for 0 and 1. And the coordinator uses a map with key chainid

Reply:
Instead, bump the starting index in the [for loop](https://github.com/cosmos/ibc-go/pull/774/files#diff-c18b7943741f428b34ce8ef78da1978fb3a372d0218b2fd5b2bd17346c7d454bR36) to 1 and make it `<= n`. It doesn't make sense that `GetChainID(3)` would return `testchain-4`

Reply:
The current solution only affects key generation and doesnt require changes on any tests.
If I change the starting index on the for loop, I will need to update every test that uses `GetChainID` to bump +1 on every call. For example every instance of `GetChainId(0)` would fail.

I can make the change if necessary.

Reply:
> It doesn't make sense that GetChainID(3) would return testchain-4

agree, I think we need to update the `for` loop @ramacarlucho

Reply:
There aren't too many instances for the bump not to be viable. The alternative option is to create a variable for the starting index, let it default to 0 and then Evmos can override it. This seems kinda unnecessary though to avoid bumping the `GetChainID` argument